packages/documentation/docs/controls/dropdown.md

```diff
@@ -38,6 +42,28 @@ frameworks={{
   javascript: SourceDropdownIcon
 }}></Playground>
 
+### Dropdown with quick actions menu
```

Comment:
Information about the verison is missing e.g. https://ix.siemens.io/docs/controls/dropdown-button